### ISSUE TITLE
How inittab is handled / Installation advice

### ISSUE THREAD

**Issue by kingdonb** [alexmyczko/livecd.gnustep.org#1]
Hello,

I'm trying to follow the installation instructions on http://www.aiei.ch/gnustep/install.txt which have a note "sorry these are not updated for 2.5"

That's OK, they are pretty good, but a few things could probably still use updating in them.

So I have lots of questions, some are probably related to this installation document needing updating, but this one has got me stumped as I transition from LiveCD to an installed VM on Sid.

How does your inittab work?

I found autostartx and I put it into my installed GNUSTEP VM with an identical inittab to yours, and it does not seem to hit autostartx.  I followed the installation advice to use `sid` distro, I went ahead and installed sysvinit according to some other instructions for getting sysvinit back in debian sid, and got rid of systemd, so it is really hitting inittab.  But I don't know what this `kk` line means, and I'm almost sure the comment above it has nothing to do with how it works.

I just want to make my installed VM boot straight to a logged-in session exactly how it works in your LiveCD.  I thought if I replicated your inittab, I would get most of the way there.  Looks like I'm still missing something.

Suggestions?  And do you want any help updating the installation document?

I would really like to have at worst a script that I can run, that does all of this configuration on a freshly installed system.  Whatever has gone into making the LiveCD "how it is" up to and including putting the orca tiff in the right place.  If such a script does exist, please publish it or point me to it!

**Comment by alexmyczko** [comment]
Do you have /etc/X11/Xwrapper.config?
```
# Xwrapper.config (Debian X Window System server wrapper configuration file)
#
# This file was generated by the post-installation script of the x11-common
# package using values from the debconf database.
#
# See the Xwrapper.config(5) manual page for more information.
#
# This file is automatically updated on upgrades of the x11-common package
# *only* if it has not been modified since the last upgrade of that package.
#
# If you have edited this file but would like it to be automatically updated
# again, run the following command as root:
#   dpkg-reconfigure x11-common
nice_value=0
allowed_users=anybody
needs_root_rights=yes
```
And is the user you use in the tty group?

**Comment by kingdonb** [comment]
I figured out the `tty` group, yes, but I did not have the correct Xwrapper.config, no

I was using
```
allowed_users=console
needs_root_rights=yes
```

Thanks!  That fixed it!

**Comment by kingdonb** [comment]
Well, now I have a great working VM and including Chrome browser and wmhdplop.

Is there anything else you think I'm still likely to have missed, after reading through the install.txt and the files in this repo?

**Comment by kingdonb** [comment]
It's working great!

**Comment by alexmyczko** [comment]
Well the /etc/skel is nice for the preconfiguration of the user. But it'll just work fine without.

**Comment by kingdonb** [comment]
What I wound up doing was a wild and crazy journey through SystemD and Kubernetes... and on debian `sid` it works great, thanks to your helpful installation advice!

But I wanted to deliver my friend an installed copy of GNUstep without any adornments, too, so what I wound up doing also is to mount a copy of the squashfs on `/mnt/live` and do `rsync -av` into `/mnt/target` so it would be identical.

Then following the instructions to configure filesystem, adding the "me" user, making sure `/etc/skel` was in place (it was), and configuring the fstab and grub.  I definitely did not forget the `/etc/skel`.  The one thing I noticed was that if you create your user as something other than `me`, the files in /etc/skel which have some references to `/home/me` will be slightly inaccurate.

But with this technique, and sticking with the `me` user, I think I was able to create a faithful copy of GNUSTEP on VirtualBox

One thing I thought would work, that ended up not working, was to simply `dd` the ISO to /dev/sda

I wound up with no bootable devices, other livecds (like elivecd, or Ubuntu) often release a hybrid form of image that can be booted directly from USB or other disks.  Would this have worked if I was using a real USB stick as a secondary boot device?  It did not appear to work on `/dev/sda`...

But it worked well to faithfully implement the instructions, except using the rsync from squashfs in place of a `cdebootstrap` of `debian/sid`.  Since I think we will prefer to work on the stable release in general.

It would be nice to have an Ansible role for doing this!
